_pages/projects.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: dashboard
+title: Projects
+---
+
+# Projects
+
+DC Government agencies are working on a variety of projects as part of the [Open Source Pilot](http://open.dc.gov/open-source-guidelines). These include websites, policy guides, documentation standards, command line tools, and more.
```

Comment:
If the guidelines site is really just guidelines, it may be somewhat confusing as a destination for the "Open Source Pilot" link. I would expect to see the projects themselves, not just instructions on how to make one.